### PR TITLE
Library paging sort

### PR DESCRIPTION
**Pull request by tjacovich**
- Updated `UserView` to explicitly sort libraries
- Added the ability to sort the library list by `date_created` and `date_last_modified` in either `asc` or `desc` order
- set default sort to `date_created` `desc`
- updated unittests

### CODE REVIEW

**Comment by tjacovich** [comment]
Relevant BeeHive PR[#200](https://github.com/adsabs/BeeHive/pull/200)

File: biblib/views/user_view.py [hunk 1]
```diff
@@ -200,17 +208,27 @@ def get(self):
...
+            
+            sort_order = get_params.get('order', default='desc', type=str)
+            if sort_order not in ['asc', 'desc']:
+                raise ValueError
+
             current_app.logger.debug("GET params: {}, start: {}, end: {}".format(get_params, start, rows))
+
         except ValueError:
-            msg = "Failed to parse input parameters: {}. Please confirm request is properly formatted.".format(request)
+            msg = "Failed to parse input parameters: {}. Please confirm request the is properly formatted.".format(request)
```
Comment (kelockhart) [inline biblib/views/user_view.py:223]: "request the" --> "the request"

File: biblib/views/user_view.py [hunk 2]
```diff
@@ -200,17 +208,27 @@ def get(self):
             get_params = request.args
             start = get_params.get('start', default=0, type=int)
             rows = get_params.get('rows', type=int)
+            
+            sort_col = get_params.get('sort', default='date_created', type=str)
+            if sort_col not in ['date_created', 'date_last_modified']: 
+                raise ValueError
+            
+            sort_order = get_params.get('order', default='desc', type=str)
```
Comment (kelockhart) [inline biblib/views/user_view.py:216]: Let's discuss this default sort order - I had thought/assumed we'd do asc by default, so the earlier results would be unchanging, but I can see the arguments for seeing the most recent libraries first instead. Let's discuss this point in person this week.

**Review by kelockhart — COMMENTED** [review]
As noted, let's discuss the defaults on the sort order before merging. Otherwise it looks good.

**Pull request merged**